cmd/metrics/metrics.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 The OpenEBS Author
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package metrics
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	ListenPort   string
+	MetricsPath  string
+	startingTime time.Time
+	// Variable for generate uptime metrics
+	Uptime = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "Uptime",
+		Help: "Uptime of node disk manager",
+	})
+)
+
+func StartHttpServer() error {
+	ListenPort = ":9090"
+	MetricsPath = "/metrics"
+	startingTime = time.Now()
```

Comment:
@ShovanMaity can these three variables be used as flags, that will be helpful in avoiding the situations where the listen port and path be occupied by some other apps.please avoid hardcoding

Reply:
@utkarshmani1997 I changed those according to your review. Please review it again.